salt/states/netconfig.py

```diff
@@ -19,16 +18,18 @@
 """
 
 # Import Salt libs
-from __future__ import absolute_import, print_function, unicode_literals
 
-import logging
```

Comment:
Leave this line and delete the try/except please.

Reply:
Done. I'm not sure why the lint check was complaining before, it accepted it this time with no issues. I also removed the import comments like you had me do in the test module.

Reply:
Awesome! Thanks!